pkg/sdk/go/certs.go

```diff
@@ -34,18 +37,13 @@ func (sdk mfSDK) IssueCert(thingID string, keyBits int, keyType, valid, token st
 	}
 
 	url := fmt.Sprintf("%s/%s", sdk.certsURL, certsEndpoint)
-	resp, err := request(http.MethodPost, token, url, d)
-	if err != nil {
-		return Cert{}, errors.NewSDKError(err)
-	}
-	defer resp.Body.Close()
-
-	if err := errors.CheckError(resp, http.StatusOK); err != nil {
-		return Cert{}, err
+	_, body, sdkerr := sdk.processRequest(http.MethodPost, url, token, string(CTJSON), d, http.StatusCreated)
```

Comment:
You should remove `request` method and replace it everywhere by `sdk.processRequest`

Reply:
On it